ui/helpers/utils/permission.js

```diff
@@ -125,28 +195,40 @@ const PERMISSION_DESCRIPTIONS = deepFreeze({
             `${coinType} (Unrecognized protocol)`}
         </span>,
       ]),
-      leftIcon: 'fas fa-door-open',
-      rightIcon: null,
+      description: t('permission_manageBip44KeysDescription', [
+        <span
+          className="tooltip-label-item"
+          key={`description-coin-type-${coinType}`}
+        >
+          {coinTypeToProtocolName(coinType) ||
+            `${coinType} (Unrecognized protocol)`}
```

Comment:
We can probably use the `unrecognizedProtocol` string?

Reply:
Yep, good catch.